packages/commerce-server/src/format-prices-for-product.ts

```diff
@@ -166,8 +173,17 @@ export async function formatPricesForProduct(
     ? appliedMerchantCoupon.percentageDiscount.toNumber() < bulkCouponPercent
     : true
 
-  const pppAvailable =
-    quantity === 1 && pppDiscountPercent > 0 && appliedMerchantCouponLessThanPPP
```

Comment:
the change looks good, my only suggestion would be to keep a secondary variable like `pppConditionsMet` with these three conditions and then that can be used below:

```javascript
const pppAvailable = userPurchases.length > 0
  ? hasPurchaseWithPPP && pppConditionsMet
  : pppConditionsMet
```

thoughts?